packages/web/hooks/use-slippage-config/index.ts

```diff
@@ -0,0 +1,148 @@
+import { useState } from "react";
+import { Dec, DecUtils, RatePretty } from "@keplr-wallet/unit";
+import { action, computed, makeObservable, observable } from "mobx";
+import { computedFn } from "mobx-utils";
+
+export class SlippageConfig {
```

Comment:
In the stores package I created a `ui-config` folder that contains mobx objects that are for storing UI state. This class would be a good fit for that (but not the hook)

Reply:
Agreed. Didn't notice that there was such folder. Will take a look.